token/doc.go

```diff
@@ -0,0 +1,25 @@
+/*
+Package token provides bearer token definition.
+
+Bearer token is attached to the object service requests, and it overwrites
+extended ACL of the container. Mainly it is used to provide access of private
+data for specific user. Therefore, it must be signed by owner of the container
+with private data.
+
+Define bearer token by setting correct lifetime, extended ACL and owner ID of
+the user that will attach token to its requests.
+	var bearerToken token.BearerToken
+	bearerToken.SetLifetime(500, 10, 10)
+	bearerToken.SetEACL(eaclTable)
+	bearerToken.SetOwner(ownerID)
+
+Bearer token must be signed by owner of the container.
+	err := bearerToken.SignToken(privateKey)
+
+Provide signed token in JSON or binary format to the request sender. Request
+sender can can attach this bearer token to the object service requests:
+	var headParams sdkClient.PrmObjectHead
```

Comment:
Lets add import since `sdkClient` is a package alias.

Reply:
Fixed.